examples/verify-example/test/upgradables.ts

```diff
@@ -0,0 +1,372 @@
+import chalk from 'chalk';
+import { Deployer } from '@matterlabs/hardhat-zksync-deploy';
+
+import * as hre from 'hardhat';
+import { Contract } from 'ethers';
+import * as zk from 'zksync-ethers';
+import { TASK_VERIFY, ZKSYNC_SEPOLIA_CHAIN_ID, useEnvironment } from './helpers';
+
+describe('verify upgradable smart contracts', async function () {
+    useEnvironment();
+
+    it('Deploy and verify box beacon', async function () {
+        const contractName = 'Box';
+        console.info(chalk.yellow(`Deploying ${contractName}...`));
+
+        const zkWallet = await hre.zksyncEthers.getWallet(0);
+
+        const deployer = new Deployer(hre, zkWallet);
+
+        const boxContract = await deployer.loadArtifact(contractName);
+        const beacon = await hre.zkUpgrades.deployBeacon(deployer.zkWallet, boxContract);
+        await beacon.waitForDeployment();
+
+        const box = await hre.zkUpgrades.deployBeaconProxy(deployer.zkWallet, await beacon.getAddress(), boxContract, [
+            42,
+        ]);
+        await box.waitForDeployment();
+
+        box.connect(zkWallet);
+        const value = await box.retrieve();
+        console.info(chalk.cyan('Box value is: ', value));
+
+        const chainId = await hre.network.provider.send('eth_chainId', []);
+        if (chainId === ZKSYNC_SEPOLIA_CHAIN_ID) {
+            const _ = hre.run(TASK_VERIFY, { address: await box.getAddress() });
+        }
+    });
+
+    it('Deploy and verify box proxy', async function () {
+        const contractName = 'Box';
+        console.info(chalk.yellow(`Deploying ${contractName}...`));
+
+        const zkWallet = await hre.zksyncEthers.getWallet(0);
+
+        const deployer = new Deployer(hre, zkWallet);
+
+        const contract = await deployer.loadArtifact(contractName);
+        const box = await hre.zkUpgrades.deployProxy(deployer.zkWallet, contract, [42], { initializer: 'store' });
+
+        await box.waitForDeployment();
+
+        box.connect(zkWallet);
+        const value = await box.retrieve();
+        console.info(chalk.cyan('Box value is: ', value));
+
+        const chainId = await hre.network.provider.send('eth_chainId', []);
```

Comment:
Please remove this check, we assume that we will use testnet for this example until local verify BE is not online.